src/Maestro/Maestro.Web/Startup.Auth.cs

```diff
@@ -62,6 +66,16 @@ private void ConfigureAuthServices(IServiceCollection services)
                                         new Claim(ClaimTypes.Role, role, ClaimValueTypes.String, GitHubScheme));
                                 }
                             },
+                            OnRemoteFailure = async context =>
+                            {
+                                var logger = context.HttpContext.RequestServices.GetRequiredService<ILogger<GitHubAuthenticationHandler>>();
+                                logger.LogError(context.Failure, "Github authentication failed.");
+                                context.HandleResponse();
+                                var res = context.HttpContext.Response;
+                                res.StatusCode = (int)HttpStatusCode.Forbidden;
+                                res.GetTypedHeaders().ContentType = new MediaTypeHeaderValue("text/html");
+                                await context.HttpContext.Response.Body.WriteAsync(Encoding.UTF8.GetBytes("403; Authentication failed. <a href='/Account/SignIn'>Sign In</a> again."));
```

Comment:
This line is really long and really indented, so it's hard to read most of it.
It's not a particularly well formed HTML document... Doesn't have a title, doesn't have the metadata, isn't going to look right...  Should we 302 them to somewhere instead, so we don't have to have inline HTML right here? And have a real page here?

Reply:
I could use a status code pages thing and redirect them to `/Error`?

Reply:
Is that easy in core land? That's probably good.

Reply:
Its a few lines of stuff, I'd have to write the Error page though.